customer-terminate-service/domain/src/main/java/se/inera/intyg/cts/domain/service/SendPackagePasswordImpl.java

```diff
@@ -23,4 +23,18 @@ public void sendPassword(Termination termination) {
       terminationRepository.store(termination);
     }
   }
+
+  /**
```

Comment:
Javadoc that doesn't add any additional value/description than the method name itself is redundant and create noise. I think your code is self-documenting.

Reply:
I prefere javadoc as best practise and have never felt that it create noise. All external libraries does contain it and i find it helpfull that you can get a small description just by howering over the classname or method. By experience self-documented methods tends to not be self-documented in the long run and/or be self documented to the one who writes them. :)